_posts/2017/12/2017-12-04-lesson-infrastructure-subcommitee.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+authors: ["Raniere Silva"]
+title: "Lesson Infrastructure Subcommittee 2017 November meeting"
+date: 2017-12-04
+time: "10:00:00"
+category: ["Community", "Lesson Infrastructure Subcommittee"]
+---
+
+On 16 Novemberr 2017 at 15:00UTC+0,
+the Lesson Infrastructure Subcommittee had their 2017 November meeting.
+This post will cover the topics discussed and their resolutions.
+
+## Software Carpentry and Data Carpentry merge
+
+With the [merge](https://software-carpentry.org/blog/2017/09/merger.html) in 2018,
+some Git repositories will be owned by a new GitHub organization.
+The Instructor Training course material already moved places,
+you can now find it at http://carpentries.github.io/instructor-training/.
+**Date for the migration will be announced next year.**
+Instructions for migrate the repository can be find [here](https://github.com/swcarpentry/styles/issues/189).
+
+## Syntax Highlight
+
+Thanks to [naught101](https://github.com/naught101),
+the next release of our lesson will offer syntax highlight to our readers.
+Lesson maintainers might need help to change
+
+````
+~~~
+print("Hello World")
+~~~
+{: .foobar}
+````
+
+to
+
+````
+~~~
+print("Hello World")
+~~~
+{: .language-foobar}
+````
+
+for example.
+If you want to help,
+send us a pull request.
+
+## Exercises Throughout the Episodes
+
+After a small discussion,
+we reach the consensus that will be better to have exercises throughout the episodes
+instead of **all** the exercises at the end of the episode.
+Lessons will migrate to the new format in a slow pace
+because this change requires a good amount of work.
+
+## Non-English Lessons
+
+If you are involved with us since 2014,
+you might remember [this post](https://software-carpentry.org/blog/2014/06/translating-software-carpentry-into-spanish.html) about the attempt to translate the lesson to Spanish
+and [this other post](https://software-carpentry.org/blog/2014/11/korean-translation.html) announcing the lessons in Korean.
+During the meeting,
+we have a conversation about the workflow to translate the lesson to other languages.
+
+Some of the conversation was archived as issues [here](https://github.com/Carpentries-ES/board/issues).
+If you want to get involved with the translation FIXME.
```

Comment:
@ErinBecker could you replace the "FIXME" at this line with the correct contact detail before merge this pull request?

Reply:
@ErinBecker is on leave @rgaiacs so there might be a slight delay with her response.

Reply:
@weaverbel Thanks for the information.

@map0logo, @agbeltran, @orchid00 can you provide the preferable contact detail?